core/templates/dev/head/editor/exploration_settings.html

```diff
@@ -431,7 +431,8 @@
 
     <h3>
       Initial parameter changes
-      <span class="glyphicon glyphicon-info-sign" tooltip="These changes are applied before the learner begins the exploration." tooltip-placement="right" style="font-size: 12px; padding-left: 18px;"></span>
+      <i class="material-icons md-18" tooltip="These changes are applied before the learner begins the exploration."
+         tooltip-placement="right" style="padding-left: 18px;">&#xE88E;</i>
```

Comment:
I would say, for this and others, please go with whatever looks good to you! The point of these changes is to make the design look nice, so I would suggest using that aim as a guide.